packages/api/orm/entity/Environment.ts

```diff
@@ -0,0 +1,22 @@
+import { Application } from "./Application";
+import { Column, Entity, ManyToOne } from "typeorm";
+import { ProvisionableEntity } from "./ProvisionableEntity";
+
+@Entity()
+export class Environment extends ProvisionableEntity {
+  @Column({ type: String, nullable: false, length: 100 })
+  name: string;
+
+  @ManyToOne(() => Application, (application) => application.environments)
+  application: Application;
+
+  // operators - simple-array or AppEnvOperator[] ?
+  @Column({ type: "simple-array", default: "" })
```

Comment:
The docs make me a little worried about this type because of the limitation that values must not contain commas. Is that a constraint we're sure we're okay with?

Reply:
I've replaced these with arrays of varchars in 6d56cf589554dff6723f2478a0caca08682a7c68
This makes them similar to `dodComponents` and `portfolioManagers` from `Portfolio`, but with a default value.